atlp-rwanda/cod-fe
# feat: user able to search [Delivers #181414575]

## Task Description
As a user, I should be able to use the search component
So that, I can easily retrieve records from both the request and approval table

## How to test this PR
- Type the keyword in the input box
- You can notice the changes in the table accordingly
- In the select option, select which type you want to search for
- click on the search icon
- results is displayed in the table

## Screenshoot
![Capture](https://user-images.githubusercontent.com/55656026/172160202-13911d9f-7d53-4e55-ae5b-33e9d4501ba3.JPG)
